scripts/_util.py

```diff
@@ -27,3 +36,33 @@ def validate_bytes(
         raise click.BadParameter("not a hex string")
 
     return decode_hex(value)
+
+
+def _account_from_keyfile(keyfile: Path, password: str) -> LocalAccount:
+    with open(keyfile, "rt") as fp:
+        privkey = Account.decrypt(json.load(fp), password)
+    return cast(LocalAccount, Account.from_key(privkey))
+
+
+def web3_for_rpc(rpc: str, account: LocalAccount) -> Web3:
+    web3 = Web3(HTTPProvider(rpc))
+
+    web3.eth.set_gas_price_strategy(rpc_gas_price_strategy)
```

Comment:
For the rollups this is probably right, but on mainnet we should use something smarter.

Reply:
I added an optional parameter